# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/IDBFactory.cpp
# Commit: 340dfa69aed8
# Full Hash: 340dfa69aed87f29b9fdeef59a650ad5eb8c1b7b
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-09-30 16:21:37
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Remove duplication in uses of IDB_LOG_MARK. r=ttung,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D46576
# ==============================================================================

diff -r a864ed2ae59a -r 340dfa69aed8 dom/indexedDB/IDBFactory.cpp
--- a/dom/indexedDB/IDBFactory.cpp	Mon Sep 30 08:54:17 2019 +0000
+++ b/dom/indexedDB/IDBFactory.cpp	Fri Sep 27 10:11:45 2019 +0000
@@ -760,17 +760,12 @@
   MOZ_ASSERT(request);
 
   if (aDeleting) {
-    IDB_LOG_MARK(
-        "IndexedDB %s: Child  Request[%llu]: "
-        "indexedDB.deleteDatabase(\"%s\")",
-        "IndexedDB %s: C R[%llu]: IDBFactory.deleteDatabase()",
-        IDB_LOG_ID_STRING(), request->LoggingSerialNumber(),
-        NS_ConvertUTF16toUTF8(aName).get());
+    IDB_LOG_MARK_CHILD_REQUEST(
+        "indexedDB.deleteDatabase(\"%s\")", "IDBFactory.deleteDatabase()",
+        request->LoggingSerialNumber(), NS_ConvertUTF16toUTF8(aName).get());
   } else {
-    IDB_LOG_MARK(
-        "IndexedDB %s: Child  Request[%llu]: "
-        "indexedDB.open(\"%s\", %s)",
-        "IndexedDB %s: C R[%llu]: IDBFactory.open()", IDB_LOG_ID_STRING(),
+    IDB_LOG_MARK_CHILD_REQUEST(
+        "indexedDB.open(\"%s\", %s)", "IDBFactory.open()",
         request->LoggingSerialNumber(), NS_ConvertUTF16toUTF8(aName).get(),
         IDB_LOG_STRINGIFY(aVersion));
   }